xml_inscriptions/transcribed/IL.Chi.UC.SAM.G.1967.115.300.xml

```diff
@@ -19,7 +19,8 @@
             <publicationStmt>
                
                 <authority>Brown University</authority> 
-                <idno type="USEpigraphy" xml:id="IL.Chi.UC.SAM.G.1967.115.300">IL.Chi.UC.SAM.G.1967.115.300</idno>
+                <idno type="USEpigraphy" xml:id="IL.Chi.UC.SMA.G.1967.115.300"
```

Comment:
Remember to change the name of the actual file if you're changing the filename throughout.